packages/sdks/snippets/gen1-react/src/routes/query-cheatsheet/queries/regex.tsx

```diff
@@ -0,0 +1,7 @@
+import { builder } from '@builder.io/react';
+
+export const getProduct = (options: { [key: string]: any }) =>
```

Comment:
similarly for all gen1 code:

```suggestion
import type { GetContentOptions } from '@builder.io/sdk';

export const getProduct = (options: GetContentOptions) =>
```

Reply:
Thank you. 🙏 I should have assumed the type would be somewhere when I couldn't find it in `@builder.io/react`. Apologies. It's been updated.